From: Donald Hunter <donald.hunter@gmail.com>
Date: Sat, 27 May 2023 14:31:04 +0100
Subject: doc: ynl: Add doc attr to struct members in genetlink-legacy spec
Patch-mainline: v6.5-rc1
Git-commit: 6d6bae63053d547e96fe4d2c9c8b4fc595bfc5ac
References: jsc#PED-3311

Make it possible to document the meaning of struct member attributes in
genetlink-legacy specs.

Signed-off-by: Donald Hunter <donald.hunter@gmail.com>
Signed-off-by: Jakub Kicinski <kuba@kernel.org>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 Documentation/netlink/genetlink-legacy.yaml |    3 +++
 1 file changed, 3 insertions(+)

--- a/Documentation/netlink/genetlink-legacy.yaml
+++ b/Documentation/netlink/genetlink-legacy.yaml
@@ -124,6 +124,9 @@ properties:
                 $ref: '#/$defs/len-or-define'
               byte-order:
                 enum: [ little-endian, big-endian ]
+              doc:
+                description: Documentation for the struct member attribute.
+                type: string
         # End genetlink-legacy
 
   attribute-sets:
